index.html

```diff
@@ -0,0 +1,168 @@
+            <ul>
+              <li><a href="./about.html">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Sponsors</a></li>
+              <li><a href="#">News</a></li>
+              <li><a href="#" id="red">EAC Campaign</a></li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+      <div class="landing">
+        <div class="window">
+          <ul class="window-list">
+            <li>
+              <a href="./about.html" class="menu-link">About</a>
+            </li>
+            <li><a href="#main-program" class="menu-link">Program</a></li>
+            <li><a href="#featured-speakers" class="menu-link">Speakers</a></li>
+          </ul>
+        </div>
+
+        <h2 class="introh2">"Hello! caring African world"</h2>
+        <div class="h1div">
+          <h1 class="introh1">EAST AFRICAN GLOBAL WARMING SUMMIT 2023</h1>
+        </div>
+        <p class="introp">
+          Coming together of three of the most affected nations within the
+          Eastern and Central African countries to explore and understand the
+          socio-economic effects of global warming to the nations.
+        </p>
+        <div class="date">
+          <p><span>2023.03.15</span>(WED) ~ <span>20</span>(SAT)</p>
+        </div>
+        <p class="location">@Kitengela, Naivash Shopping Center, Nairobi</p>
+      </div>
+    </header>
+    <section class="main-program" id="main-program">
+      <h1>Main Program</h1>
+      <div class="underline"></div>
+      <div class="program-container">
+        <div class="card">
+          <img src="./assets/lecture.png" alt="lectur icon" />
+          <h2>Talks</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, sint!
+          </p>
+        </div>
+        <div class="card">
+          <img src="./assets/networking.png" alt="networking icon" />
+          <h2>Connect</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, sint!
+          </p>
+        </div>
+        <div class="card">
+          <img src="./assets/group (2).png" alt="community icon" />
+          <h2>Forum</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, sint!
+          </p>
+        </div>
+        <div class="card">
+          <img src="./assets/visitors.png" alt="vspeakers icon" />
+          <h2>Exhibit</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, sint!
+          </p>
+        </div>
+        <div class="card">
+          <img src="./assets/workshop.png" alt="workshop icon" />
+          <h2>Links</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, sint!
+          </p>
+        </div>
+      </div>
+      <div class="join-button">Join ECA community</div>
+      <p class="see-prog">SEE THE WHOLE PROGRAM</p>
```

Comment:
- [ ] Hello @Adholah96, great effort working on the main program design as suggested by the previous reviewer, but yet your design looks different from what we have in the design. I would like to make some suggestion below on what to change in your design to make it looks perfect and in line with what we have in the design.

  - In the `.main-program` class, kindly change ``padding: 1.3rem 4rem;`` to ``padding: 1.3rem 0;``
  - In the `.card` class, kindly add `width: 90%;`, remove `margin-bottom: 1.2rem` and replace it with `margin: 1rem auto` also replace the padding with `padding: 1rem`.
  - In the `.card p` class, kindly add a `margin-left: 30px` and remove the `padding`

With all the above changes, I believe your main project cards will look perfect and in line with the design template.

Your Design. 👇 

![adloh](https://user-images.githubusercontent.com/78634841/217362212-d16acd85-cb30-40a1-834e-a7dfd556089a.png)

Reply:
Wow  @brainconnect93  🙌👏. Thank you so much... this solved it and I appreciate you taking the time to show me this.✔